controllers/common-controller-utils_test.go

```diff
@@ -196,45 +230,70 @@ func expectedRamenSecretData() map[string][]byte {
 
 func TestMirrorPeerSecretReconcile(t *testing.T) {
 	cases := []struct {
+		testCaseID      int
```

Comment:
no need for ID. `name` should be enough.

Reply:
ack